shard.yml

```diff
@@ -4,11 +4,11 @@ version: 0.4.6
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
 
-crystal: 0.36.1
+crystal: '>= 0.36.1, < 2.0.0'
 
 license: MIT
 
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: 0.13.4
+    version: 0.14.1
```

Comment:
This needs to stay as 0.13.4. [See Ameba Issue](https://github.com/crystal-ameba/ameba/issues/208).

Reply:
Oh thank you for letting me know! 🙏

Reply:
Reverted in https://github.com/luckyframework/habitat/pull/70/commits/8f7ca12501859c58495541c0997ab659f7f00612!